modules/virt-about-workload-updates.adoc

```diff
@@ -2,26 +2,53 @@
 //
 // * virt/upgrading-virt.adoc
 
+:_content-type: CONCEPT
 [id="virt-about-workload-updates_{context}"]
 = About workload updates
 
-* When you upgrade {VirtProductName}, virtual machine workloads including `libvirt`, `virt-launcher`, and `qemu` update automatically if they support live migration.
-+
+When you upgrade {VirtProductName}, virtual machine workloads, including `libvirt`, `virt-launcher`, and `qemu`, update automatically if they support live migration.
```

Comment:
Does "upgrade" need to be changed to "update" here?

Reply:
Good catches! Thanks!